### ISSUE TITLE
What is the recommended annotation format? 

### ISSUE THREAD

**Issue by andanders** [milesial/Pytorch-UNet#380]
I'm considering if I should annotate my dataset with COCO, and then convert polygons to RLE.
The example caravan data for this project uses .gif masks, and I can convert RLE to binary images.
However, would the easiest solution be to import COCO.json annotation data straight in when training? Is that even an option?
My main proglem is that I don't know how to convert a polygon to RLE. I would prefer not to write some rasterization script myself (coco->gif).

**Comment by andanders** [comment]
I will use the COCO option iscrowd.

**Comment by andanders** [comment]
I'm back... I've tried all these annotation tools, and none of them seem to support RLE/iscrowd. What am I missing here?
https://www.makesense.ai/
https://github.com/scalabel/scalabel
https://www.robots.ox.ac.uk/~vgg/software/via/
https://github.com/wkentaro/labelme
https://github.com/tzutalin/labelImg

**Comment by milesial** [comment]
At the end of the day you need a targets tensor with your target classes in it. How you do it is up to you.
You can modify the dataloading to support json data from COCO, or preprocess the json into images or numpy/torch pickle files before the training.
If you didn't find any tool to do that, my guess is that you would have to do the conversion yourself
